Makefile

```diff
@@ -47,21 +47,32 @@ help:
 	@echo "  logs-join                  to join xdist log files into one"
 	@echo "  logs-clean                 to delete all xdist log files in the root"
 	@echo "  pyc-clean                  to delete all temporary artifacts"
+	@echo "  uuid-check                 to check for duplicated or empty @id: in testimony docstring tags"
+	@echo "  uuid-fix                   to fix all duplicated or empty @id: in testimony docstring tags"
+	@echo "  can-i-push?                to check if local changes are suitable to push"
+	@echo "  install-commit-hook        to install pre-commit hook to check if changes are suitable to push"
+	@echo "  gitflake8                  to check flake8 styling only for modified files"
+	@echo "  clean-cache                to clean pytest cache files"
+	@echo "  clean-all                  to clean cache, pyc, logs and docs"
 
 docs:
 	@cd docs; $(MAKE) html
 
 docs-clean:
+	$(info "Cleaning docs...")
 	@cd docs; $(MAKE) clean
 
-test-docstrings:
+test-docstrings: uuid-check
+	$(info "Checking for errors in docstrings and testimony tags...")
 	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/api
 	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/cli
 	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/rhci
 	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/ui
 	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/rhai
+	testimony $(TESTIMONY_OPTIONS) validate tests/foreman/sys
```

Comment:
This directory does not exists in 6.2.z, or we remove this line, or we pick `tests/foreman/sys` to 6.2.z